modules/ducktests/tests/ignitetest/utils/ignite_test.py

```diff
@@ -40,3 +44,45 @@ def monotonic():
             so that only the difference between the results of consecutive calls is valid.
         """
         return monotonic()
+
+    def copy_ignite_root_dir(self):
+        """
+        Copying root directory from service nodes to the results directory.
+        """
+        for service in self.test_context.services:
```

Comment:
Do we really need this feature?
What if we'll have 1T of data of each node at a specific test?
In case you need to investigate, just use docked and you"ll have an access to everything.

Reply:
And all data will be wiped away by another test. We should move it to somewhere, or collect somewhere, if test failed

Reply:
she is useful
for it to work, we must explicitly call it in the test
docker after test cleans up working directory

Reply:
@ivandasch sound strange to run something else when you debugging specific test.
@Sega76 you may disable the cleanup. I don't like the idea to have such uberfeatures.

Reply:
i think the disable cleaning it does not make sense to, when all tests are started, cleaning will be when starting the next test, I shoud is more correct to collect the data when the test fails